rct229/ruletest_engine/ruletest_jsons/section5/rule_5_49.json

```diff
@@ -0,0 +1,352 @@
+                        "id": "RMR 1",
+                        "buildings": [
+                            {
+                                "id": "Building 1",
+                                "building_open_schedule": "Required Schedule 1",
+                                "building_segments": [
+                                    {
+                                        "id": "Building Segment 1",
+                                        "zones": [
+                                            {
+                                                "id": "Zone 1",
+                                                "thermostat_cooling_setpoint_schedule": "Cooling Schedule 1",
+                                                "thermostat_heating_setpoint_schedule": "Heating Schedule 1",
+                                                "infiltration": {
+                                                    "id": "Infiltration 1",
+                                                    "infiltration_flow_rate": 0.5285811363839998
+                                                },
+                                                "terminals": [
+                                                    {
+                                                        "id": "Terminal 1",
+                                                        "served_by_heating_ventilation_air_conditioning_system": "HVAC 1"
+                                                    }
+                                                ],
+                                                "surfaces": [
+                                                    {
+                                                        "id": "Surface 1",
+                                                        "adjacent_to": "EXTERIOR",
+                                                        "tilt": 90,
+                                                        "area": 464.51519999999994
+                                                    },
+                                                    {
+                                                        "id": "Surface 1",
+                                                        "adjacent_to": "GROUND",
+                                                        "tilt": 90,
+                                                        "area": 464.51519999999994
+                                                    }
+                                                ]
+                                            }
+                                        ],
+                                        "heating_ventilation_air_conditioning_systems": [
+                                            {
+                                                "id": "HVAC 1",
+                                                "cooling_system": {
+                                                    "id": "Cooling System 1",
+                                                    "sensible_cool_capacity": 0.0
```

Comment:
Could you check if ```sensible_cool_capacity``` shouldn't be 0? When it's 0, there is an error message saying ```Construction``` is missing.